integration-tests/tests/common/mod.rs

```diff
@@ -807,15 +807,15 @@ impl<'a> TestServerConfigurator<'a> {
                         "user",
                         users::get_current_username().unwrap().to_str().unwrap(),
                     );
-                    cfg.insert("sshkey", "ssh-ed25519 sshkey_default user@example.com");
+                    cfg.insert("sshkey", "ssh-ed25519 sshkey_default user@example.com;ssh-ed25519 sshkey_default user@example2.com");
                     cfg.insert("password", "testpassword");
                 } else {
                     L.l("per_device_serviceinfo is set, using device specific values");
                     cfg.insert(
                         "user",
                         users::get_current_username().unwrap().to_str().unwrap(),
                     );
-                    cfg.insert("sshkey", "ssh-ed25519 sshkey_per_device user@example.com");
+                    cfg.insert("sshkey", "ssh-ed25519 sshkey_per_device user@example.com;ssh-ed25519 sshkey_default user@example2.com");
```

Comment:
this inserts a string in the configuration file that does not match what a user would do:
```yaml
service_info:
  initial_user:
    username: testuser
    password: testpassword
    sshkeys:
    - ssh-ed25519 sshkey_per_device user@example.com;ssh-ed25519 sshkey_default user@example2.com
```

we are defining a [list](https://github.com/fedora-iot/fido-device-onboard-rs/blob/main/util/src/servers/configuration/serviceinfo_api_server.rs#L81C1) in there, so something like the following would be desirable:
```yaml
service_info:
  initial_user:
    username: testuser
    password: testpassword
    sshkeys:
    - ssh-ed25519 sshkey_per_device user@example.com
    - ssh-ed25519 sshkey_default user@example2.com
```